pkg/printutil/printutil.go

```diff
@@ -136,3 +170,40 @@ func strSliceToInterSlice(ss []string) []interface{} {
 	}
 	return is
 }
+
+func (t *Table) createHeaderPadding() {
+	if len(t.headerPadding) == 0 {
+		for _, num := range t.Header {
+			colLength := len(num) + 5
+			t.headerPadding = append(t.headerPadding, colLength)
+		}
+	}
+}
+
+func (t *Table) coalesceHeaderAndRowPadding(val int, i int) {
+	if val < t.headerPadding[i] {
+		t.altPadding[i] = t.headerPadding[i]
+	} else {
+		t.headerPadding[i] = t.altPadding[i]
+	}
+}
+
+func (t *Table) dynamicPadding(rows []TempRow) {
+	for _, row := range rows {
+		for i, col := range row.Values {
+			colLength := len(col) + 5
+			if len(t.altPadding) != len(row.Values) {
+				t.altPadding = append(t.altPadding, colLength)
+			} else {
+				if t.altPadding[i] < colLength {
```

Comment:
Remove nested if statement in favor of another top-level `if` or an `else if`, depending on exact needs

Reply:
Fixed.